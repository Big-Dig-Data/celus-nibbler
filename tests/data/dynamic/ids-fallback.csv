Title,ID,Platform,YOP,Metrics,2020-01,2020-02,2020-03
"T1","1111-1111","P1","2015","Sessions",1,2,3
"T1","1111-1111","P1","2015","Denied",0,1,0
"T2","978-3-16-148410-0","P1","2013","Sessions",4,5,6
"T2","978-3-16-148410-1","P1","2013","Denied",3,0,2
"T3","","P1","2018","Denied",2,8,6
"T4","80-902734-1-6","P1","2018","Sessions",3,9,7
"T4","80-902734-1-7","P1","2018","Denied",4,0,8
