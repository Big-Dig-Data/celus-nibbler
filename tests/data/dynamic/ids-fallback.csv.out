"2020-01-01","2020-01-31","","T1","Sessions","Platform:P1|YOP:2015","Print_ISSN:1111-1111","1"
"2020-02-01","2020-02-29","","T1","Sessions","Platform:P1|YOP:2015","Print_ISSN:1111-1111","2"
"2020-03-01","2020-03-31","","T1","Sessions","Platform:P1|YOP:2015","Print_ISSN:1111-1111","3"
"2020-01-01","2020-01-31","","T1","Denied","Platform:P1|YOP:2015","Print_ISSN:1111-1111","0"
"2020-02-01","2020-02-29","","T1","Denied","Platform:P1|YOP:2015","Print_ISSN:1111-1111","1"
"2020-03-01","2020-03-31","","T1","Denied","Platform:P1|YOP:2015","Print_ISSN:1111-1111","0"
"2020-01-01","2020-01-31","","T2","Sessions","Platform:P1|YOP:2013","ISBN:978-3-16-148410-0","4"
"2020-02-01","2020-02-29","","T2","Sessions","Platform:P1|YOP:2013","ISBN:978-3-16-148410-0","5"
"2020-03-01","2020-03-31","","T2","Sessions","Platform:P1|YOP:2013","ISBN:978-3-16-148410-0","6"
"2020-01-01","2020-01-31","","T2","Denied","Platform:P1|YOP:2013","ISBN:978-3-16-148410-0","3"
"2020-02-01","2020-02-29","","T2","Denied","Platform:P1|YOP:2013","ISBN:978-3-16-148410-0","0"
"2020-03-01","2020-03-31","","T2","Denied","Platform:P1|YOP:2013","ISBN:978-3-16-148410-0","2"
"2020-01-01","2020-01-31","","T3","Denied","Platform:P1|YOP:2018",,"2"
"2020-02-01","2020-02-29","","T3","Denied","Platform:P1|YOP:2018",,"8"
"2020-03-01","2020-03-31","","T3","Denied","Platform:P1|YOP:2018",,"6"
